plugin-common/src/main/kotlin/com/gradleup/gr8/Gr8Task.kt

```diff
@@ -50,6 +62,16 @@ abstract class Gr8Task : DefaultTask() {
     mapping.disallowChanges()
   }
 
+  fun javaLauncher(launcher: JavaLauncher) {
+    javaLauncher.set(launcher)
+    javaLauncher.disallowChanges()
+  }
+
+  fun toolchain(toolchain: JavaToolchainSpec) {
+    javaLauncher.set(javaToolchainService.launcherFor(toolchain))
+    javaLauncher.disallowChanges()
+  }
```

Comment:
added this for convenience, but technically from the plugin perspective it's not strictly necessary